packages/react-notifications/src/components/notifier/notifier.js

```diff
@@ -22,6 +22,7 @@ const Notifier = props => {
       // Remove notification when component "unmounts"
       notification.dismiss();
     };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []); // run only once
```

Comment:
Here the eslint rule only shows a warning

<img width="616" alt="image" src="https://user-images.githubusercontent.com/1110551/62030045-7380ee00-b1e4-11e9-8fad-7482b226c2d7.png">

But I do only want to run this once...

Reply:
Hm, Then I would add a comment and disable the rule so nobody confuses it as a mistake?

Reply:
Yeah...but do you think it makes sense to keep the empty array?

Reply:
Yeah. Otherwise changing any prop would show another notification, right? I don't think that's intended. Also I think before this used hooks it was just `componentDidMount` and no `compnentDidUpdate` which triggered another notification?

Reply:
Yes exactly